src/pages/editor/curators/AddCuratorForm.tsx

```diff
@@ -160,14 +160,17 @@ export const AddCuratorForm = ({
                 ? values.roles.filter((selectedRole) => selectedRole.rolename !== role)
                 : [...values.roles, { type: 'Role', rolename: role }];
 
-              if (isRemovingRole) {
+              if (!isRemovingRole) {
+                if (role === RoleName.CuratorThesisEmbargo) {
+                  newRoles = [
+                    ...values.roles,
+                    { type: 'Role', rolename: RoleName.CuratorThesis },
+                    { type: 'Role', rolename: RoleName.CuratorThesisEmbargo },
+                  ];
+                }
+              } else if (isRemovingRole) {
```

Comment:
Trenger ikke å sjekk om `!condition` om man allerede er i else for sjekken til `condition`

```suggestion
              } else {
```

Reply:
Ofte bittelitt mer lettlest om `true` condition er først, men det er kanskje mer en smakssak. Slipper en `!` da